### DISCUSSION TITLE
Dockerhub RDF4J-Workbench 4.1.1?

### DISCUSSION THREAD

**Discussion by maxstolze** [eclipse-rdf4j/rdf4j#4160] [General]
Will there be a RDF4J-Workbench 4.1.1 on Dockerhub any time soon?

**Comment by barthanssens** [comment]
Hi @maxstolze 

sure, I'll push a new version later this evening / tomorrow morning

Best regards

Bart

**Reply by barthanssens** [reply]
Should be available now

**Reply by maxstolze** [reply]
Thank you! This was fast!
